completions/ip

```diff
@@ -48,7 +48,7 @@ _ip()
                 COMPREPLY=($(compgen -W "$c" -- "$cur"))
                 ;;
             *)
-                COMPREPLY=($(compgen -W "help $($1 help 2>&1 | command sed -e \
+                COMPREPLY=($(compgen -W "help $($1 -c=never help 2>&1 || $1 help 2>&1 | command sed -e \
```

Comment:
This is not correct; both `$1` calls need to be wrapped in `{ ... }` like `{ $1 -c=never help || $1 help; } 2>&1 || command ...`